TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -122,6 +122,10 @@ public static void CheckPicture([NotNull] CommonDataDescriptionAndDataRedefines
 
             //Store validation result for future usages
             codeElement.PictureValidationResult = pictureValidationResult;
+            if (codeElement.DataType == null || codeElement.DataType == DataType.Unknown)
```

Comment:
It seems odd to have the 2 possibilities. Maybe keep the original assignments to Unknown value and assert the DataType is not null here ?

Reply:
I changed to only check for `codeElement.DataType == DataType.Unknown`, as `codeElement.DataType` cannot return null.